app/src/main/java/org/simple/clinic/summary/PatientSummaryScreenController.kt

```diff
@@ -464,7 +464,12 @@ class PatientSummaryScreenController @Inject constructor(
               .map { { ui: Ui -> ui.showUpdatePhoneDialog(patientUuid) } }
         }
 
-    val showForMissingPhone = screenCreations
+    val waitTillABpIsRecorded = events
```

Comment:
The test has to be updated to take this change into account. Right now, it expects the dialog to be shown when the screen is created.

Reply:
So weird, the tests passed on my local machine. Maybe I missed something. Fixing this.